src/backend/environment.ts

```diff
@@ -15,49 +15,61 @@ const isDev = () => {
 // Se på disse url-ene. Her lenker dev og env til hele lenken, mens den siste har bare port 8082 og sender med resten i api-service
 
 export interface EnvironmentProps {
-  loginService: string;
+  wonderwallUrl: string;
   apiUrl: string;
+  apiProxyUrl: string;
+  dokumentProxyUrl: string;
   dokumentUrl: string;
-  mergeDokumentUrl: string;
   port: number;
   dekoratørEnv: 'dev' | 'prod';
+  oauthCallbackUri: string;
 }
 
 const getEnv = (): EnvironmentProps => {
   if (isProd()) {
     return {
-      loginService: 'https://loginservice.nav.no/login?',
+      wonderwallUrl:
+        'https://familie.nav.no/familie/alene-med-barn/soknad/oauth2/login?redirect=',
+      apiProxyUrl: 'https://www.nav.no/familie/alene-med-barn/ettersending/api',
       apiUrl: 'https://www.nav.no/familie/alene-med-barn/soknad-api',
-      dokumentUrl:
-        'https://www.nav.no/familie/dokument/api/mapper/familievedlegg',
-      mergeDokumentUrl:
-        'https://www.nav.no/familie/dokument/api/mapper/merge/familievedlegg',
+      dokumentProxyUrl:
+        'https://www.nav.no/familie/alene-med-barn/ettersending/dokument',
+      dokumentUrl: 'https://www.nav.no/familie/dokument',
```

Comment:
Her burde vi bruke service discovery url'er (også for familie-ef-soknad?)

Reply:
I fix. Byttet egentlig dette i soknad til å bruke service discovery etter tilbakemelding fra deg, men ser det er to steder som url'ene listes opp der (miljø.ts og Environment.ts), noe som er kilde til forvirring og som vi kan rydde opp i egen PR på soknad